src/main/java/core/basesyntax/service/impl/CsvFormatReportGenerator.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Comparator;
+import java.util.Map;
+
+public class CsvFormatReportGenerator implements ReportGenerator {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String CSV_SEPARATOR = ",";
+    private static final String START_MESSAGE = "fruit,quantity";
```

Comment:
Let's make names more informative and obvious
```suggestion
    private static final String COMMA = ",";
    private static final String HEADER_LINE = "fruit,quantity";
```